### PR TITLE
Mobile Appointments List

### PR DESCRIPTION
**Pull request by kreek**
## Description of change
Adds `GET /mobile/v0/appointments` which returns a list of VA and Community Care appointments ordered by UTC appointment date. VA appointments do not include location data so a call to the facilities api adds that data when needed. Final implementation required updates to the VA and CC adapters so those classes and specs have been updated as well.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#14853

### CODE REVIEW

**Review by jperk51 — APPROVED** [review]
LGTM

**Comment by kreek** [comment]
Closing as this PR seems to consistently run into spec errors in another module, multiple reruns have not gotten past the following errors which I've been told occur 'randomly'
```
1) VBA Document Uploads Endpoint #create /v1/uploads returns a UUID and location
   Failure/Error: expect(response).to have_http_status(:accepted)
     expected the response to have status code :accepted (202) but it was :internal_server_error (500)


2) VBA Document Uploads Endpoint #create /v0/uploads returns a UUID and location
   Failure/Error: expect(response).to have_http_status(:accepted)
     expected the response to have status code :accepted (202) but it was :internal_server_error (500)
```

File: Gemfile.lock [hunk 1]
```diff
@@ -105,7 +105,6 @@ PATH
 
 GEM
   remote: https://rubygems.org/
-  remote: https://enterprise.contribsys.com/
```
Comment (erluti) [inline Gemfile.lock:108]: Make sure the sidekiq enterprise gems aren't removed

Comment (kreek) [reply Gemfile.lock]: Not sure why that happened because I have the enterprise env var set

**Pull request merged**